docs/Developers/Development_Setup.rst

```diff
@@ -17,48 +17,57 @@ Here we will be showing how to have a virtualenv using :code:`venv` and
 of the standard library and requires no extra installation. However,
 you can use whichever you find suitable to yourself.
 
-Using venv
-~~~~~~~~~~
+Using virtualenv
+~~~~~~~~~~~~~~~~
 
 - Make sure to have Python 3 installed in your local machine.
 
-- Setting up virtualenv with venv :
-    ::
+- As we have to make a virtual environment with Python >= 3.4 .
 
-        $ cd working_dir # move into the dir where you want to create coala-venv
-        $ python3 -m venv coala-venv
-        # This creates a isolated python environment called coala-venv
-        # in your current directory.
-        # To activate the environment type:
-        $ source coala-venv/bin/activate
-        # To exit the environment simply type:
-        (coala-venv)$ deactivate
+- Here is a demo how it should be done.
 
-- Now you can activate the environment and start
-  `the next part <https://coala.io/devsetup#installing-from-git>`_.
+- Setting up virtualenv with name coala-venv :
 
-Using virtualenv
-~~~~~~~~~~~~~~~~
+::
+
+        $ cd working_dir
+
+-NOTE:
+ If you have both python 3 and python 2 installed check this out first.
+
+::
+
+	$ virtualenv coala-venv -p $(which python3)
 
-- Install virtualenv using pip :
-    ::
+-This creates a isolated python environment called coala-venv
+ In your current directory.
+ As coala only works for Python >= 3.4.
+ By using $ which python3
+ We can know the path where python3 is installed.
 
-        $ pip install virtualenv
+::
 
-- Create the virtualenv :
-    ::
+    	$ source coala-venv/bin/activate
+    	(coala-venv)$ python --version
```

Comment:
@NiklasMM  should I change what gitmate bot said or it's all right.
According to me my this PR was the best it has no issues and should be merged .
Please merge it as it would be my first merged code ,this is critical for submitting my already written proposal.